docs/source/verifiers.rst

```diff
@@ -39,7 +39,7 @@ are given below:
 
 If you want verification, you should try with the ``ChecksumTableVerifier``
 first if you're copying whole tables at a time. If that takes too long, you can
-try using the ``IterativeVerifier``. Alternatively, you can verify in a staging
+try using the ``IterativeVerifier`` or ``InlineVerifier``. Alternatively, you can verify in a staging
 run and not verify during the production run (see :ref:`copydbinprod`).
 
 IterativeVerifier
```

Comment:
Can we add a line saying this is deprecated and will be removed.

Reply:
done!